inc/saihash.h

```diff
@@ -54,120 +54,123 @@ typedef enum _sai_native_hash_field_t
      * in case for encapsulated packets.
      * Used for both IPv4 and IPv6
      */
-    SAI_NATIVE_HASH_FIELD_DST_IP,
+    SAI_NATIVE_HASH_FIELD_DST_IP = 0x00000001,
 
     /** Native hash field inner source IP */
-    SAI_NATIVE_HASH_FIELD_INNER_SRC_IP,
+    SAI_NATIVE_HASH_FIELD_INNER_SRC_IP = 0x00000002,
 
     /** Native hash field inner destination IP */
-    SAI_NATIVE_HASH_FIELD_INNER_DST_IP,
+    SAI_NATIVE_HASH_FIELD_INNER_DST_IP = 0x00000003,
 
     /**
      * @brief Native hash field source IPv4.
      *
      * Also, refers to the outer source IPv4
      * in case for encapsulated packets
      */
-    SAI_NATIVE_HASH_FIELD_SRC_IPV4,
+    SAI_NATIVE_HASH_FIELD_SRC_IPV4 = 0x00000019,
```

Comment:
How will this work for any upgrade from a previous version ?

Reply:
warm boot? will not be binary compatible with previous versions like v1.6